src/Support/ScheduledTasks/ScheduleMonitoringConfigurationsRepository.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Spatie\ScheduleMonitor\Support\ScheduledTasks;
+
+class ScheduleMonitoringConfigurationsRepository
+{
+    private array $store = [];
```

Comment:
Let's add a docblock that shows the content of this array.

Alternatively you could introduce a `MonitoredScheduledTask` class and put that in the array